EndlessClient/ControlSets/LoggedInControlSet.cs

```diff
@@ -46,6 +53,7 @@ protected override void InitializeControlsHelper(IControlSet currentControlSet)
             _changePasswordButton = GetControl(currentControlSet, GameControlIdentifier.ChangePasswordButton, GetPasswordButton);
             _characterInfoPanels.AddRange(_characterInfoPanelFactory.CreatePanels(_characterSelectorProvider.Characters));
 
+            _allComponents.Add(new CurrentUserInputTracker(_endlessGameProvider, _userInputRepository));
```

Comment:
One thing to test is that this doesn't cause a second `CurrentUserInputTracker` to be added to the game's components when transitioning to the `PlayingTheGame` game state. I think this is handled in `GameStateActions.RemoveOldComponents` but it would be good to double-check. If it works as expected then this should be fine.

Reply:
I didn’t dig too deep into how the control set stuff worked. I assumed that the components were isolated to the login state and would clear when changing state. I’ll see if I can test that

Reply:
Confirmed that the components do get removed when changing control sets
<img width="712" alt="devenv_PPXFlfnnMy" src="https://user-images.githubusercontent.com/3664832/160277191-9e19ca8e-67e3-4294-b5f7-c2a377051758.png">
<img width="1250" alt="devenv_KJCq8JIpOC" src="https://user-images.githubusercontent.com/3664832/160277193-37830f85-bcad-413a-a1e5-48a16bfa4c99.png">